server/pxf-jdbc/src/main/java/org/greenplum/pxf/plugins/jdbc/JdbcResolver.java

```diff
@@ -130,109 +129,81 @@
     @Override
     public OneRow setFields(List<OneField> record) throws UnsupportedOperationException, ParseException {
         int column_index = 0;

```

Comment:
Rename to columnIndex?
```suggestion
        int columnIndex = 0;
```